### PR TITLE
Document usage with STORAGES setting

### PR DESCRIPTION
**Pull request by jwygoda**
No code changes are required to make django-storages compatible with the new setting.

Resolves https://github.com/jschneier/django-storages/issues/1207

### CODE REVIEW

File: docs/backends/amazon-S3.rst [hunk 1]
```diff
@@ -18,16 +18,29 @@ Settings
...
+    STORAGES = {"default": "storages.backends.s3boto3.S3Boto3Storage"}
+
+
 To allow ``django-admin collectstatic`` to automatically put your static files in your bucket set the following in your settings.py::
 
+    # django < 4.2
     STATICFILES_STORAGE = 'storages.backends.s3boto3.S3StaticStorage'
 
+    # django >= 4.2
+    STORAGES = {"default": "storages.backends.s3boto3.S3StaticStorage"}
```
Comment (jschneier) [inline docs/backends/amazon-S3.rst:34]: This should be `"staticfiles"`.

File: docs/backends/amazon-S3.rst [hunk 2]
```diff
@@ -18,16 +18,29 @@ Settings
...
+
 If you want to use something like `ManifestStaticFilesStorage`_ then you must instead use::
 
+    # django < 4.2
     STATICFILES_STORAGE = 'storages.backends.s3boto3.S3ManifestStaticStorage'
 
+    # django >= 4.2
+    STORAGES = {"default": "storages.backends.s3boto3.S3ManifestStaticStorage"}
```
Comment (jschneier) [inline docs/backends/amazon-S3.rst:42]: This should be `"staticfiles"`.

**Review by pauloxnet — APPROVED** [review]

**Comment by pauloxnet** [comment]
@jschneier this PR seems mergable to mee.

**Comment by jschneier** [comment]
Test failures are unrelated and due to other changes, merging.

**Comment by jschneier** [comment]
Thanks for the PR.

**Pull request merged**